database/factories/ModelFactory.php

```diff
@@ -16,6 +18,25 @@
 |
 */
 
+// Event Factory
+$factory->define(Event::class, function (Generator $faker) {
+    return [
+        'northstar_id' => str_random(24),
+        'submission_type' => 'signup',
+    ];
+});
+
+// Signup Factory
+$factory->define(Signup::class, function (Generator $faker) {
+    return [
+        'campaign_id' => $this->faker->randomNumber(4),
+        'campaign_run_id' => $this->faker->randomNumber(4),
+        'event_id' => function() {
+            return factory(Event::class)->create()->id;
+        },
+    ];
+});
+
 // Reportback Factory
```

Comment:
Ahhh, yeah! 💥

I think the problem is that there's no `signup_id` set on the embedded event, so when the "has many" relationship tries to look for events with that column, it can't find any.